common/riff/chunk.ksy

```diff
@@ -0,0 +1,17 @@
+meta:
+  id: chunk
+  encoding: ASCII
+  endian: le
+seq:
+  - id: id
+    type: str
+    size: 4
```

Comment:
In general, it's a bad idea to have chunk 4-byte type identifiers to be handles as strings:

* String conversion is comparatively slow and inefficient. All production parsers will use an 4-byte integer enum here, while KS-generated parser will be slower due to string usage / allocation / deallocation / string comparisons.
* Strings have encoding, and in general case, there is no guarantee that it will be anything actually readable by human. Quite a few RIFF-based formats use arbitary byte sequences (like `0xdeadc0de`), which is neither valid UTF-8, nor valid ASCII.